src/components/TextInput/TextInput.stories.tsx

```diff
@@ -29,47 +18,143 @@ export const Default: Story = {
   args: {
     id: 'Default',
     name: 'Default',
-    placeholder: 'Placeholder text',
-    width: 'default',
+    value: 'Default',
     type: 'text'
   }
 };
 
+export const Hover: Story = {
+  render: _arguments => <TextInput {..._arguments} className='hover' />,
+  name: 'Hover',
+  args: {
+    ...Default.args,
+    value: 'Hover'
+  }
+};
+
+export const Focus: Story = {
+  render: _arguments => <TextInput {..._arguments} className='focus' />,
+  name: 'Focus',
```

Comment:
Let's remove the `name` param when not needed.

![Screenshot 2023-12-01 at 10 15 00 AM](https://github.com/cfpb/design-system-react/assets/2592907/3eb2a861-beb0-4964-8801-7871462d3e09)
```suggestion
```